tests/dummy/app/routes/users/list.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+import PaginatedRouteMixin from  'ember-osf/mixins/paginated-route';
+
+export default Ember.Route.extend(AuthenticatedRouteMixin, PaginatedRouteMixin, {
+    store: Ember.inject.service(),
```

Comment:
Note to self: in a separate PR, go through routes and clean up any place where `store` is being injected manually into the route. (this boilerplate is unnecessary)

Ditto for `session` if the route doesn't explicitly use it.